apps/product/src/main/java/com/ecommerce/product/model/Category.java

```diff
@@ -0,0 +1,18 @@
+package com.ecommerce.product.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
```

Comment:
```suggestion
@SuperBuilder
```

Add the `SuperBuilder` annotation let you have access to `builder()` method